Some modifications to maintain compatibility with the latest TWiLight Menu++

Ta180m opened the pull request:
Starting with v6.8.3 of TWiLight Menu++, DS game saves are stored in a separate `saves` folder than the DS ROMs, so I made a few small modifications to maintain compatibility with the latest TWiLight Menu++.

Commit:
Update main.cpp

Updated save path

Commit:
Create README.md

Commit:
Update README.md

Commit:
build

Commit:
Merge branch 'master' of https://github.com/Ta180m/Make-Forwarder-Dsi

Commit:
Update README.md

Commit:
Update README.md

Commit:
Update README.md

Commit:
Attempt to fix icon

Commit:
Merge branch 'master' of https://github.com/Ta180m/Make-Forwarder-Dsi

Commit:
Hopefully fixed icon

Commit:
Update README.md

Commit:
Cleaned up build

Commit:
Merge branch 'master' of https://github.com/Ta180m/Make-Forwarder-Dsi

Commit:
Update README.md

Commit:
Create .gitignore

FreeUnblockedGames commented:
https://github.com/Ta180m/Make-Forwarder-Dsi
This person already did that. Maybe try click "Fork" Before creating a pull request?

spellboundtriangle committed:
Remove 2 gigabyte SD card requirement

The latest hiyaCFW automatically handles creation of dummy files to account for the "limit", so this is not necessary.

Commit:
Merge pull request #1 from spellboundtriangle/spellboundtriangle-no2gb

Remove 2 gigabyte SD card requirement

RonkZeDonk committed:
allow more than one savefile

Commit:
Merge pull request #2 from RonkZeDonk/savefiles

Allow more than one save file

Commit:
Add note about self-compiling
